redwood/.env.defaults

```diff
@@ -32,3 +32,5 @@ INFURA_IPFS_SECRET=7089dd9b3331c0ec6d824a1f589e2fe9
 SMTP_HOST=smtp.ethereal.email
 SMTP_USER=irving.rau62@ethereal.email
 SMTP_PASSWORD=m7ADZ6TuH1mbbVvV4r
+
+ETHERSCAN_API_KEY=YOUR_ETHERSCAN_API_KEY
```

Comment:
Since this is a non-working example it should go in .env.example, not .env.defaults